src/test/kotlin/com/atlassian/performance/tools/virtualusers/SimpleWebdriverScenario.kt

```diff
@@ -0,0 +1,84 @@
+package com.atlassian.performance.tools.virtualusers
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.virtualusers.api.browsers.CloseableRemoteWebDriver
+import com.atlassian.performance.tools.virtualusers.api.browsers.HeadlessChromeBrowser
+import com.atlassian.performance.tools.virtualusers.api.config.VirtualUserTarget
+import com.atlassian.performance.tools.virtualusers.api.diagnostics.WebDriverDiagnostics
+import com.atlassian.performance.tools.virtualusers.lib.api.Scenario
+import java.util.concurrent.atomic.AtomicInteger
+
+// TODO show that RTE can be disabled via setup
+class SimpleWebdriverScenario(
+    virtualUserTarget: VirtualUserTarget,
+    private val meter: ActionMeter
+) : Scenario(
+    virtualUserTarget, meter
+) {
+    private val driver: CloseableRemoteWebDriver = HeadlessChromeBrowser().start()
```

Comment:
Constructors shouldn't do real work or heavy-lifting like starting processes.
I might be able to create an instance, but never reach `before` or `getActions` (due to an `if` or a `throw`).